.storybook/preview.tsx

```diff
@@ -71,7 +72,7 @@ addDecorator(InjectGlobalStyles);
 const ADDONS_REQUIRED_IN_OPTIONS = {
   isFullscreen: false,
   isToolshown: true,
-  panelPosition: 'right',
+  panelPosition: 'bottom',
```

Comment:
Pretty sure this is the standard, and with all the addons this causes less crowding on the canvas.